### ISSUE TITLE
Tweakable/patternable latency/quantum

### ISSUE THREAD

**Issue by yaxu** [tidalcycles/Tidal#951]
The new link sync requires changes in boottidal.hs to tweak the quantum and latency.

It'd be good if this could be tweaked and ideally patterned live. There is `nudgeAll` but I think this still only works by adding latency, and not shifting back in term (it works, but only within the latency buffer).

**Comment by Zalastax** [comment]
I agree about quantum and beats per cycle being good if it could be changed live.

Latency is harder as there may be many targets.

It would be possible to make nudge affect how far ahead calculation is performed. That way, nudging a lot would not require a reboot of Tidal.
